### PR TITLE
Interpreter: fix NaN handling in FMA instructions

### PR DESCRIPTION
**Pull request by Tilka**
~~Will eventually fix [issue 8364](https://dolp.in/i8364).~~ Removed the Jit64 stuff for now.

### CODE REVIEW

**Comment by Tilka** [comment]
This seems like it could use a game-specific option (default off). Thoughts?

File: Source/Core/Core/PowerPC/Interpreter/Interpreter_FPUtils.h [hunk 1]
```diff
@@ -84,7 +84,7 @@ inline double Force25Bit(double d)
 // these functions allow globally modify operations behaviour
 // also, these may be used to set flags like FR, FI, OX, UX
 
-inline double NI_mul(const double a, const double b)
+inline double NI_mul(double a, double b)
```
Comment (BhaaLseN) [inline Source/Core/Core/PowerPC/Interpreter/Interpreter_FPUtils.h]: Any particular reason to drop the ```const``` on those and the other ones?

Comment (Tilka) [inline Source/Core/Core/PowerPC/Interpreter/Interpreter_FPUtils.h]: It's just that it doesn't really serve any particular purpose.

**Comment by FioraAeterna** [comment]
I think a game-specific option for hyper-accurate floating point emulation could be appropriate. Not sure if we should split it up for maximum performance (NaNs, FPSCR, other things...?) or if we should just put it all in one.

**Comment by Tilka** [comment]
Added an option "AccurateNaNs" which is on by default for easy testing.

**Comment by JMC47** [comment]
Ping?  Is this ready to merge?

**Comment by Tilka** [comment]
No, and it probably shouldn't be merged for 5.0. (The FMA-unrelated commits have already been merged as part of other PRs.)

**Comment by Tilka** [comment]
I've dropped the Jit64 stuff for now, because that will still take a bit longer. This PR should be ready to review/merge.

**Pull request merged**